src/Medology/Behat/StoreContext.php

```diff
@@ -249,12 +250,34 @@ public function assertThingPropertyContains(string $thing, string $property, str
         }
     }
 
+    /**
+     * Parses each TableNode cell for references to stored items and replaces them with the value from the store.
+     *
+     * @param TableNode $table a table node containing the values to parse
+     *
+     * @throws InvalidArgumentException if the string references something that does not exist in the store
+     * @throws InvalidArgumentException if callable $onGetFn does not take exactly one argument
+     * @throws InvalidArgumentException if callable $hasValue does not take exactly two arguments
+     * @throws InvalidArgumentException if callable $onGetFn returns something other than array, object or callable
+     * @throws OutOfBoundsException     if the specified stored item does not have the specified property or
+     *                                  key
+     * @throws ReflectionException      if $onGetFn was provided but the reflection API says it does not exist. This
+     *                                  should not be possible.
+     * @throws ReflectionException      if $hasValue was provided but the reflection API says it does not exist. This
+     *                                  should not be possible.     */
+    public function injectStoredValuesTable(TableNode $table): array
+    {
+        return array_values(array_map(function ($row) {
+            return array_map(function ($value) {
+                return $this->injectStoredValues($value);
+            }, $row);
+        }, $table->getRows()));
```

Comment:
It seems you missed the second part of my [previous suggestion](https://github.com/Medology/FlexibleMink/pull/415#discussion_r767805228).

```suggestion
        return array_values(array_map(function ($row) {
            return array_map([$this->storeContext, 'injectStoredValues'], $row);
        }, $table->getRows()));
```